src/build.ts

```diff
@@ -149,6 +164,9 @@ export async function build(
       effects.output.write(`${faint("bundle")} ${path} ${faint("→")} `);
       const clientPath = getClientPath(path === "/_observablehq/client.js" ? "index.js" : path.slice("/_observablehq/".length)); // prettier-ignore
       const define: {[key: string]: string} = {};
+      if (path === "/_observablehq/stdlib/duckdb.js") {
+        define["process.DUCKDB_MANIFEST"] = JSON.stringify(duckdb_manifest);
```

Comment:
Seems a little strange to use `process.DUCKDB_MANIFEST` as the magic here. Like either we’d pretend it’s an environment variable as `process.env.DUCKDB_MANIFEST` or just make it a global `DUCKDB_MANIFEST`.